src/sentry/crons/consumers/checkin.py

```diff
@@ -0,0 +1,61 @@
+import logging
+from typing import Mapping
+
+from arroyo.backends.kafka.consumer import KafkaPayload
+from arroyo.processing.strategies.abstract import ProcessingStrategy, ProcessingStrategyFactory
+from arroyo.processing.strategies.commit import CommitOffsets
+from arroyo.processing.strategies.run_task import RunTask
+from arroyo.types import Commit, Message, Partition
+from django.db import transaction
+
+from sentry.models import CheckInStatus, Monitor, MonitorCheckIn, MonitorStatus
+from sentry.utils import json
+from sentry.utils.dates import to_datetime
+
+logger = logging.getLogger(__name__)
+
+
+def process_message(message: Message[KafkaPayload]) -> None:
+    params = json.loads(message.payload.value)
```

Comment:
We'll delegate all validation to Relay and assume these are validated payloads. Esp the Serializer is no longer needed if Relay handles all the mutating endpoints.